index.html

```diff
@@ -16,7 +16,67 @@
 </head>
 <body>
   <header>
+
+    <div class="header-div">
+      <div class="head-box-text">Code Journal </div>
+    </div>
   </header>
+
+
+
+
+
+  <div class="container">
+
+
+      <div class="form-div" data-view="entry-form">
+
+        <h2 class="form-h2 margin-left-form">New Entry</h2>
+
+        <form id="codeform">
+          <div class="row">
+
+            <div class=" column-half">
+              <img src="images/placeholder-image-square.jpg" alt="placeholder" class="placeholder-img rounded">
+            </div>
+
+              <div class="column-half">
+                <div >
+
+                  <div class="form-item-div input-bottom-margin">
+                    <label for="title" class="form-label">Title</label><br>
+                    <input type="type" name="title" id="title" class="rounded" required>
+                  </div>
+
+                  <div class="form-item-div">
+                    <label for="photo-url" class="form-label">Photo URL</label><br>
+                    <input type="type" name="photo-url" id="photo-url" class="photoUrl rounded" required>
+                  </div>
+
+                </div>
+             </div>
+
+            </div>
+
+
+          <div class="form-item-div">
+            <label for="notes" class="form-label margin-left-form">Notes</label><br>
+             <textarea class="column-full margin-left-form"  id="notes" name="notes" rows="5" cols="33" required></textarea>
+
+          </div>
+
+          <div>
+             <button type="submit" form="codeform" class="rounded" >SAVE</button>
+          </div>
```

Comment:
This should also be a row -> column -> button sort of setup to fall in line with proper layout class usage